common/views/components/MediaAnnotations/MediaAnnotations.tsx

```diff
@@ -0,0 +1,53 @@
+import { ReactElement, FunctionComponent } from 'react';
```

Comment:
Worth putting this in Cardigan?

Reply:
Don't think so it basically renders a [DownloadLink](https://cardigan.wellcomecollection.org/?path=/story/components--downloadlink)